localization/sparse_mapping/src/vocab_tree.cc

```diff
@@ -356,7 +501,7 @@ void MatDescrToVec(cv::Mat const& mat, std::vector<float> * vec) {
   (*vec).reserve(mat.cols);
   (*vec).clear();
   for (int c = 0; c < mat.cols; c++) {
-    float val = static_cast<float>(mat.at<uchar>(0, c));
+    float val = static_cast<float>(mat.at<float>(0, c));
```

Comment:
Is this conditional on the vocab using surf vs brisk features?

Reply:
Yes. It's an overloaded function. The one that takes BriefDescriptor has uchar. I removed the unnecessary static cast.